src/phoenix/trace/dsl/filter.py

```diff
@@ -40,15 +43,18 @@ class AliasedAnnotationRelation:
 
     index: int
     name: str
+    annotation_type: str
     table: AliasedClass[models.SpanAnnotation] = field(init=False, repr=False)
     _label_attribute_alias: str = field(init=False, repr=False)
     _score_attribute_alias: str = field(init=False, repr=False)
 
     def __post_init__(self) -> None:
         table_alias = f"span_annotation_{self.index}"
-        alias_id = f"{randint(0, 10**6):06d}"  # prevent conflicts with user-defined attributes
+        # alias_id = f"{randint(0, 10**6):06d}"
+        alias_id = uuid4().hex
```

Comment:
is there a reason why a longer suffix is preferred?

Reply:
for me was less about length and just about using something that's conventionally a unique id

Reply:
This change is just replacing a shorter random number with a longer random number, so either way would suffice for the purpose of avoiding collision with user attributes. By the way, if you don't disagree with the intent here, can you un-delete the comment that describes it? But just for my understanding, is the concern here that `randint` is confusing because it's unconventional for this use case?

Reply:
yeah, that's the only reason on my end!